style.css

```diff
@@ -119,25 +262,17 @@ nav > a {
   gap: 12px;
   width: 327px;
   height: 552px;
-  background: #ffffff;
+  background: #fff;
   border: 1px solid #dfe1e6;
   border-radius: 16px;
   margin-top: 114px;
-
-}
-
-.imagen-contenedor {
-  display: flex;
-  width: 295px;
-  height: 220px;
-  border-radius: 8px;
 }
 
 .tonic > h1 {
   display: flex;
   width: 295px;
   height: 36px;
-  font-family: 'Poppins';
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] I decided to highlight this line because this references the part I am making my comment on in your index.html Below screenshot will show you the part of the code I am requesting the changes on. Kindly apply the required changes in all the work section when applicable.
 -  It is required to have only one `h1` tag on a webpage. Any other subtitle should use the remaining heading aside the `h1` (h2, .....h6). Kindly change all the underlined element from `h1` to any of the  `hx` heading.
 
### Project title
![your-h1](https://github.com/santitoot/finish-mobile-version/assets/26597432/bc53af89-c3f8-4c2b-a6f2-01f1d2c29d0d)

- [ ] It is also required that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul `and each of them a `li` Kindly use unorder list to group your project technologies list and each list should be inside `li`.

### Your Project tech list
![your-language-list](https://github.com/santitoot/finish-mobile-version/assets/26597432/181a66f2-2603-4dbf-ba3d-2860d786c76c)